akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/MaxThroughputSpec.scala

```diff
@@ -174,7 +197,8 @@ object MaxThroughputSpec extends MultiNodeConfig {
       case EndResult(totalReceived) ⇒
         val took = NANOSECONDS.toMillis(System.nanoTime - startTime)
         val throughput = (totalReceived * 1000.0 / took)
-        println(
+
+        reporter.reportResults(
           s"=== MaxThroughput ${self.path.name}: " +
             f"throughput ${throughput * testSettings.senderReceiverPairs}%,.0f msg/s, " +
             f"${throughput * payloadSize * testSettings.senderReceiverPairs}%,.0f bytes/s (payload), " +
```

Comment:
The previous inaccuracy was due to warmup messages spilling into the totalReceived number. I fixed it now by wrapping warmup messages so they can be ignored for counting. (It might have a little impact on warmup as the code-paths won't be exactly the same).